packages/twenty-front/src/modules/object-record/record-table/components/RecordTableBodyFetchMoreLoader.tsx

```diff
@@ -30,15 +32,25 @@ export const RecordTableBodyFetchMoreLoader = () => {
     onChange: onLastRowVisible,
   });
 
+  const StyledText = styled.div`
+    align-items: center;
+    box-shadow: none;
+    color: ${grayScale.gray40};
+    display: flex;
+    height: 32px;
+    padding-left: 40px;
+  `;
+
   return (
-    <tbody ref={tbodyRef}>
```

Comment:
we should keep the tbody with the enclosing ref (it needs to be there in the DOM while isFetchingMore is false)

Reply:
You can still style the tbody as you wish :)